plejd/ble.bluez.js

```diff
@@ -379,8 +377,20 @@ class PlejdService extends EventEmitter {
     this.characteristics.lastData.StartNotify();
   }
 
+  async throttledInit(delay) {
+    if(this.delayedInit){
```

Comment:
Code style mainly, but may I suggest initializing `this.delayedInit` (and possibly other new class variables) in the constructor?

Reply:
Absolutely, I just tried stuff out at 2:30 in the night!